doc/release-notes/6.3-release-notes.md

```diff
@@ -0,0 +1,453 @@
+
+### Additional Facet Settings
+
+Extra settings have been added giving an instance admin more choices in selectively limiting the availability of search facets on the Collection and Dataset pages. 
+
+See the [Disable Solr Facets](https://guides.dataverse.org/en/6.3/installation/config.html#DisableSolrFacets) sections of the Config Guide for more info.
+
+### Hidden HTML Fields
+
+External Controlled Vocabulary scripts, configured via [:CVocConf](https://guides.dataverse.org/en/6.3/installation/config.html#cvocconf), can now access the values of managed fields as well as the term-uri-field for use in constructing the metadata view for a dataset.
+
+Those values are hidden and can be found with the html attribute `data-cvoc-metadata-name`.
+
+For more information, see [#10503](https://github.com/IQSS/dataverse/pull/10503).
+
+[↑ Table of Contents](#table-of-contents)
+
+## Bug Fixes
+
+### Re-Addition of ‘Cell Counting’ to Life Science Block
+
+Re-adding value `cell counting` to Life Science metadata block's Measurement Type vocabularies accidentally removed in `v5.1`. 
+
+### Incomplete Metadata Bug Fix
+
+Bug fixed for the ``incomplete metadata`` label being shown for published dataset with incomplete metadata in certain scenarios. This label will now be shown for draft versions of such datasets and published datasets that the user can edit. This label can also be made invisible for published datasets (regardless of edit rights) with the new option ``dataverse.ui.show-validity-label-when-published`` set to `false`.
+
+### Identical Role Error Message
+
+An error is now correctly reported when an attempt is made to assign an identical role to the same collection, dataset, or file. #9729 #10465
+
+### OpenAPI Re-Enabled
+
+In Dataverse 6.0 when Payara was updated it caused the url `/openapi` to stop working:
+
+- https://github.com/IQSS/dataverse/issues/9981
+- https://github.com/payara/Payara/issues/6369
+
+Additionally to the fix of the URL on this release we are also making some changes on how we provide the OpenAPI document:
+
+When it worked in Dataverse 5.x, the `/openapi` output was generated automatically by Payara, but in this release we have switched to OpenAPI output produced by the [SmallRye OpenAPI plugin](https://github.com/smallrye/smallrye-open-api/tree/main/tools/maven-plugin). This gives us finer control over the output.
+
+For more information, see the section on  [OpenAPI](https://guides.dataverse.org/en/6.3/getting-started.html#openapi) in the API Guide.
+
+### Math Challenge Fixed on 403 Error Page
+
+On forbidden access error page, also know as 403 error page, the math challenge is now correctly display to submit the contact form.
+
+### Ingest Option Bug Fixed
+
+A bug that prevented the Ingest option in the File page Edit File menu from working has been fixed
+
+[↑ Table of Contents](#table-of-contents)
+
+## Persistence
+[↑ Table of Contents](#table-of-contents)
```

Comment:
This is empty and is no longer part of the table of contents above. Remove?

Reply:
"Persistence" was removed. Resolving.